lib/pymedphys/_gamma/implementation/shell.py

```diff
@@ -17,9 +17,10 @@
 
```

Comment:
I haven't added myself to the copyright. It feels like too small a change for that. But it occurs to me that I should learn the rules for this

Reply:
Small by what measure?
Lines of code?
Adding a thousand lines of code that doesn't accomplish much is not a huge contribution.
Changing two lines of code and improving performance by more than 2x... that's a huge contribution.

Reply:
Add it in Matt 🙂

Reply:
Add an extra line on the top, where it's just your copyright with the year 2022. If I do anything to this code in 2022 then I can add my name to yours 🙂

Reply:
Cool! Thanks! Will do.